step.yml

```diff
@@ -53,3 +53,16 @@ inputs:
       value_options:
         - "yes"
         - "no"
+  - cache_level: "all"
+    opts:
+      category: Options
+      title: Set the level of cache
+      is_required: true
+      description: |-
+        Sets the level of cache.
```

Comment:
please include this line in the summary

Reply:
summary is one of our standard, the reason behind is that without the summary there is no any info provided for the user in the WFE unless they click on the input to expand the description